.semaphore/generate-semaphore-yaml.sh

```diff
@@ -8,8 +8,11 @@ for out_file in semaphore.yml semaphore-scheduled-builds.yml; do
   cat semaphore.yml.d/02-global_job_config.yml >>$out_file
   cat semaphore.yml.d/03-promotions.yml >>$out_file
 
+  # use sed to properly indent blocks
   echo "blocks:" >>$out_file
-  cat semaphore.yml.d/blocks/*.yml >>$out_file
+  cat semaphore.yml.d/blocks/*.yml | sed -e 's/^./  &/' >>$out_file
+
+  cat semaphore.yml.d/99-after_pipeline.yml >>$out_file
```

Comment:
this file is technically not a block, so moving it out of `blocks/` so it has the correct indentation